angel_system/interfaces/emotion_models.py

```diff
@@ -0,0 +1,75 @@
+import abc
```

Comment:
@maxlchen are these emotion files something we want in this PR? These were in the forked repo when I started and I suspect the answer is no.